wiki/People/Users_with_unique_titles/id.md

```diff
@@ -1,18 +1,14 @@
----
-outdated: true
----
+# Daftar pengguna dengan gelar khusus
 
-# Pengguna dengan gelar unik
+Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [gugus pengguna (*user group*)](/wiki/People/The_Team#user-groups). Meskipun demikian, terdapat beberapa pengguna yang secara khusus dianugerahi gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang telah mereka torehkan.
 
-Gelar pengguna biasanya untuk mereka yang berada dalam [kelompok pengguna](/wiki/People/The_Team#user-groups), namun khusus untuk pengguna yang tercantum di bawah ini, mereka memiliki gelar karena alasan lain. Sebagian besar gelar ini merupakan bentuk penghargaan terhadap kontribusi atau prestasi yang telah dicapai, selain itu sebagai sebuah ikon atau salah satu momen bersejarah lucu di osu!
-
-## Berdasarkan kategori, dengan alasannya
+## Berdasarkan kategori, beserta alasannya
 
 ### Pemenang World Cup
 
-Salah satu hadiah yang diberikan kepada pemenang [World Cup resmi](/wiki/Tournaments#official-world-cups) adalah gelar pengguna yang berlaku selama satu tahun semenjak mereka memenangkan turnamen itu.
+Setiap tahunnya, osu! menganugerahkan para pemenang turnamen [World Cup](/wiki/Tournaments#official-world-cups) dengan gelar khusus yang berhak untuk mereka sandang selama kurang lebih satu tahun hingga penyelenggaraan turnamen World Cup berikutnya.
 
-Pemenang [OWC 2020](/wiki/Tournaments/OWC/2020) dengan gelar **osu! Champion**:
+Berikut merupakan para pemenang [OWC 2020](/wiki/Tournaments/OWC/2020) yang saat ini menyandang gelar **osu! Champion**:
```

Comment:
```suggestion
Berikut-berikut di bawah ini merupakan para pemenang [OWC 2020](/wiki/Tournaments/OWC/2020) yang saat ini memiliki gelar **osu! Champion**:
```

Reply:
(this applies to your other suggestions that are similar to this btw)

what you're suggesting atm, `Berikut-berikut di bawah ini`, grammatically speaking is actually ***very*** wrong. in english what you're suggesting is equivalent to `The followings below names are the winners...` which just doesn't make proper sense altogether.

`Berikut merupakan para pemenang` is already correct as is as it corresponds to the much more plausible `The following are the winners of...`.

Reply:
yeah, kinda missed about that there